sites/fast-website/rollup.config.js

```diff
@@ -1,7 +1,7 @@
 import alias from "@rollup/plugin-alias";
 import path from "path";
 import commonJS from "rollup-plugin-commonjs";
-import ejs from "rollup-plugin-ejs";
+import jst from "rollup-plugin-jst";
```

Comment:
Is this related to the windows validation issues?

Reply:
yes. When we imported jst, it did fixed some issues on windows validation